File: Unit1/Array/traversal.java
package Unit1.Array;

// Array : An Array is a Linear data structure 
// It is a collection of elements of same type 
// stored in contiguous memory locations.

// Defining a class 'traversal' with main method to traverse an
// array using for loop and printing its elements one by one

public class traversal {
    public static void main(String args[]) {
        int arr[] = { 10, 20, 30, 40, 50 };

        System.out.println("Array elements are : ");
        for (int i = 0; i < arr.length; i++) {
            System.out.println(arr[i]);
        }
    }
}

File: Unit1/Array/traversal.c
// Array : A linear data structure that stores elements of the
// same data type in contiguous memory locations.

#include <stdio.h>
int main()
{
    // Get the number of elements from the user and declare an array of that size
    int size;
    printf("Enter the size of Array: ");
    scanf("%d", &size);
    int arr[size];

    // Input Array Elements from the user
    printf("Enter the Array Elements: ");
    for (int i = 0; i < size; i++)
    {
        scanf("%d", &arr[i]);
    }

    // Displaying the entered array elements
    printf("Array Elements are: \n");
    for (int i = 0; i < size; i++)
    {
        printf("%d\n", arr[i]);
    }
}
